packages/wow-ui/src/components/ActionSheet/ActionSheet.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import type { Meta, StoryFn } from "@storybook/react";
+
+import ActionSheet from "@/components/ActionSheet";
+import Box from "@/components/Box";
+import Button from "@/components/Button";
+import useOpenState from "@/hooks/useOpenState";
+
+const meta = {
+  title: "UI/ActionSheet",
+  component: ActionSheet,
+  tags: ["autodocs"],
+  parameters: {
+    componentSubtitle: "액션시트 컴포넌트",
+    a11y: {
+      config: {
+        rules: [{ id: "color-contrast", enabled: false }],
+      },
+    },
+    docs: {
+      description: {
+        component:
+          "ActionSheet.Header, ActionSheet.Body, ActionSheet.Footer를 조합해서 쓸 수 있어요.\n" +
+          "1. **Header**: 액션시트의 헤더로, Text, SubText를 포함할 수 있습니다.\n" +
+          "2. **Body**: 액션시트의 본문으로, 필요에 맞게 컨텐츠를 포함할 수 있습니다.\n" +
+          "3. **Footer**: 액션시트의 푸터로, 버튼 또는 버튼 묶음을 포함할 수 있습니다.",
+      },
+    },
+  },
+  argTypes: {
+    isOpen: {
+      description: "액션시트의 표시 여부를 설정합니다.",
+      control: {
+        type: "boolean",
+      },
+    },
+    onClose: {
+      description: "액션시트를 닫는 함수입니다.",
+      control: false,
+    },
+    style: {
+      description: "액션시트의 커스텀 스타일을 설정합니다.",
+      table: {
+        type: { summary: "CSSProperties" },
+        defaultValue: { summary: "{}" },
+      },
+      control: false,
```

Comment:
p4;
```suggestion
      control: {type : 'object'},
```
style도 type 을 지정해서 컨트롤 할 수 있으면 좋을 거 같아요!!